src/components/families/FamilyTable.tsx

```diff
@@ -106,10 +107,26 @@ const FamilyTable = ({
           childrenInfo += ", ";
         }
       });
+
+      const enrolment = current_enrolment || {
+        preferred_class: { name: "N/A" },
+        enrolled_class: { name: "N/A" },
+        status: EnrolmentStatus.UNASSIGNED,
+      };
+      const enrolledClassName = enrolment.enrolled_class
+        ? enrolment.enrolled_class.name
+        : "N/A";
+      const preferredClassName = enrolment.preferred_class
+        ? enrolment.preferred_class.name
+        : "N/A";
       const familyRow: FamilyTableRow = {
         [DefaultFieldKey.FIRST_NAME]: parent.first_name,
         [DefaultFieldKey.LAST_NAME]: parent.last_name,
         [DefaultFieldKey.CHILDREN]: childrenInfo,
+        [DefaultFieldKey.STATUS]: enrolment.status,
+        [DefaultFieldKey.ENROLLED]: current_enrolment ? "True" : "False",
+        [DefaultFieldKey.CURRENT_CLASS]: enrolledClassName,
+        [DefaultFieldKey.CURRENT_PREFERRED_CLASS]: preferredClassName,
         ...args,
       };
```

Comment:
maybe a little cleaner to prevent duplicate logic?

```suggestion
      const enrolment = current_enrolment || {
        enrolled_class: null,
        preferred_class: null,
        status: EnrolmentStatus.UNASSIGNED,
      };
      const familyRow: FamilyTableRow = {
        [DefaultFieldKey.FIRST_NAME]: parent.first_name,
        [DefaultFieldKey.LAST_NAME]: parent.last_name,
        [DefaultFieldKey.CHILDREN]: childrenInfo,
        [DefaultFieldKey.ENROLLED]: current_enrolment ? "True" : "False",
        [DefaultFieldKey.CURRENT_CLASS]: enrolment.enrolled_class
          ? enrolment.enrolled_class.name
          : "N/A",
        [DefaultFieldKey.CURRENT_PREFERRED_CLASS]: enrolment.preferred_class
          ? enrolment.preferred_class.name
          : "N/A",
        [DefaultFieldKey.STATUS]: enrolment.status,
        ...args,
      };
```